mu4/plugins/api/enums.h

```diff
@@ -21,6 +21,7 @@
 #define __PLUGIN_API_ENUMS_H__
 
 #include <QQmlPropertyMap>
+#include <QMetaEnum>
```

Comment:
Is this header used anywhere in this file?

Reply:
@RomanPudashkin 
Yes. It needed to build.
There is below in this file:
`return new Enum(QMetaEnum::fromType<T>(), parent);`